tests/phpunit/tests/block-template-utils.php

```diff
@@ -187,6 +187,28 @@ function test_inject_theme_attribute_in_block_template_content() {
 		$this->assertEquals( $content_with_no_template_part, $template_content );
 	}
 
+	function test_remove_theme_attribute_in_block_template_content() {
```

Comment:
```suggestion
	/**
	 * @ticket 54448
	 */
	function test_remove_theme_attribute_in_block_template_content() {
```

:1234: Each of the new tests will need a ticket reference and docblock.

Could this use a data provider too so an earlier assertion failing doesn't prevent the following assertions from running.

Reply:
Thanks for pointing this out. I will change the test to use a data provider.